solutions/search/rag/playground-context.md

```diff
@@ -20,13 +20,16 @@ Context is the information you provide to the LLM, to optimize the relevance of
 There are a few ways to optimize this context for better results. Some adjustments can be made directly in the Playground UI. Others require refining your indexing strategy, and potentially reindexing your data.
 
 ::::{note} 
-Currently you can only select **one field** to be provided as context to the LLM.
+{applies_to}`stack: preview 9.0` Currently you can only select **one field** to be provided as context to the LLM.
```

Comment:
"currently" is misleading here, I would just use a note with applies_to information:

https://elastic.github.io/docs-builder/syntax/admonitions/#applies-to-information